package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.11.9",
   "main": "dist/sim/index.js",
   "dependencies": {
+    "edmonds-blossom": "^1.0.0",
```

Comment:
I assume @Zarel will prefer to wait to merge this until you reimplement this dependency in-house.

Reply:
In the chat-dev channel on discord, Zarel said the dependency was fine.